test/parallel/test-buffer-copy.js

```diff
@@ -23,7 +23,7 @@ let cntr = 0;
   b.fill(++cntr);
   c.fill(++cntr);
   const copied = c.copy(b, 0, 0);
-  assert.strictEqual(c.length, copied);
+  assert.strictEqual(copied, c.length);
   for (let i = 0; i < c.length; i++) {
     assert.strictEqual(c[i], b[i]);
```

Comment:
I think these should be swapped as well (we copied `c` into `b`, so `c` holds the expected values?)

Reply:
:+1: